src/renderers/native/ReactNativeStack.js

```diff
@@ -53,6 +44,25 @@ var ReactNative = {
   /* eslint-enable camelcase */
 
   unmountComponentAtNodeAndRemoveContainer: ReactNativeMount.unmountComponentAtNodeAndRemoveContainer,
+
+  __SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED: {
+    // Used for Flow types
+    SyntheticEvent: require('SyntheticEvent'),
```

Comment:
Yup, this is on the Quip doc checklist. Initially I saw SE was being required a dozen or so places internally but then realized it was a built-in type and just haven't ticked off that item yet. Thanks for pointing it out though! 😁